upi/openstack/02_network.yaml

```diff
@@ -156,7 +168,8 @@
     command:
       cmd: "openstack floating ip set --port {{ os_port_ingress }} {{ os_ingress_fip }}"
 
-  - name: 'Set MTU for the first neutron network'
+
+  - name: 'Create SNAT subnet'
```

Comment:
Apologies that i suggested adding this, but since this is not per cluster, so we need to do this only once when the external network is created, and not part of this playbook. We have to do it once when we create the external network. Let's remove this.

Reply:
Done!